crowbar_framework/app/models/service_object.rb

```diff
@@ -1725,4 +1741,38 @@ def active_update(proposal, inst, in_queue, bootstrap = false)
       [400, e2.message]
     end
   end
+
+  def skip_unready_nodes(bc, inst, new_elements, old_elements)
+    skip_unready_nodes_config = Rails.application.config.experimental["skip_unready_nodes"]
```

Comment:
The check earlier on is much safer (with all the `fetch` magic). Also... it's questionable if we need the check in both places ;-)

Reply:
ah indeed, I should remove this now as its useless!